shard.go

```diff
@@ -25,7 +25,7 @@ type cacheShard struct {
 	stats Stats
 }
 
-func (s *cacheShard) get(key string, hashedKey uint64) ([]byte, error) {
+func (s *cacheShard) getWrappedEntry(key string, hashedKey uint64) ([]byte, error) {
```

Comment:
This function should be after `get` and `getWithInfo`

Reply:
done